lua/lvim/plugins.lua

```diff
@@ -3,7 +3,6 @@ local commit = {
   cmp_buffer = "a706dc69c49110038fe570e5c9c33d6d4f67015b",
   cmp_luasnip = "16832bb50e760223a403ffa3042859845dd9ef9d",
   cmp_nvim_lsp = "134117299ff9e34adde30a735cd8ca9cf8f3db81",
```

Comment:
```suggestion
  cmp_nvim_lsp = "134117299ff9e34adde30a735cd8ca9cf8f3db81",
  cmp_nvim_lua = "d276254e7198ab7d00f117e88e223b4bd8c02d21",
```

Reply:
It seems OK to keep that, as we don't have tabnine but have it there too.